modules/navigation.js

```diff
@@ -0,0 +1,14 @@
+const links = document.querySelectorAll('.link');
+const slides = document.querySelectorAll('.slide');
+
+export default function navigate() {
```

Comment:
- [ ] As the previous reviewer specified, It would be better if you refactor your function definition to use arrow syntax 😄 

e.g. 

```
const navigate = () =>  {...content}

export default navigate